src/Globe/Globe.js

```diff
@@ -187,9 +186,11 @@ Globe.prototype.setLayerVisibility = function(id, visible) {
     this.layersConfiguration.setLayerVisibility(id, visible);
 
     var cO = function(object) {
```

Comment:
ok  done! but i did not push my code yet because i dont know what branch i'm now

@mlamure  is branch name is still Wfs2_rebased? i will push my code and can you please take a look on your code because i did not see your points and line any more